edx_repo_tools/repo_access_scraper/README.rst

```diff
@@ -0,0 +1,36 @@
+Repo Access Scraper
+###################
+
+This tool records who is granted write (or admin) access to a repo.  It writes a .md file with teams and people who have write access to each repo. It also captures screenshots of the GitHub access settings pages for each involved repo and team.
+
+#. Create a Python 3.8 virtualenv.
+
+#. Install repo-tools (https://github.com/edx/repo-tools) into your virtualenv::
+
+   $ python -m pip install /path/to/repo-tools
+
+#. Install the extras for this tool::
+
+   $ python -m pip install -r requirements/repo_access_scraper.txt
```

Comment:
Isn't there a way to do this with the extras in setup.py?

Reply:
That is an interesting idea.  I'm not sure I know how to integrate our "make upgrade" flow with extras in setup.py.